packages/utils/src/keystoreKeyGen.js

```diff
@@ -1,27 +1,28 @@
-import secp256k1 from 'secp256k1';
-import keystoreCrypto from './keystoreCrypto';
+const secp256k1 = require('secp256k1');
+const keystoreCrypto = require('./keystoreCrypto');
 
-export default {
-  /**
-   * Generate public key
-   *
-   * @param password
-   * @param v3Keystore
```

Comment:
tested on `auth`, `vault`, `connect` repos - all is fine, 

but founded two errors after install `core/utils` to `core/class`

```bash
\endpass-core\packages\class>yarn tsc                                                                                              
yarn run v1.19.1                                                                                                                                     $ 
\endpass-core\packages\class\node_modules\.bin\tsc                                                                               
node_modules/@endpass/utils/keystoreKeyGen.d.ts:1:60 - error TS2304: Cannot find name 'V3Keystore'.                                                                                                                                                                                                       
1 export function getPublicKey(password: string, v3Keystore: V3Keystore): Buffer;                                                                                                                                 
~~~~~~~~~~                                                                                                                                                                                                                                   
node_modules/@endpass/utils/keystoreKeyGen.d.ts:2:61 - error TS2304: Cannot find name 
'V3Keystore'.                                                                                                                                                                                                       
2 export function getPrivateKey(password: string, v3Keystore: V3Keystore): Buffer;                                                                                                                                 
~~~~~~~~~~                                                                                                                                                                                                                                                                                                                                                                                       
Found 2 errors.                                                                                                                                                                                                                                                                                           
error Command failed with exit code 1.                                                                                                               
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.                
```